config/samples/assisted-installer/example-core-standard.yaml

```diff
@@ -0,0 +1,436 @@
+##################################
```

Comment:
Same here. If there is nothing unique we don't need a "core" example. If there is content we can update the generic example with, that is good.